exercises/resistor-color/description.md

```diff
@@ -15,4 +15,6 @@ These colors are encoded as follows:
 
 Mnemonics map the colors to the numbers, that, when stored as an array, happen to map to their index in the array: Better Be Right Or Your Great Big Values Go Wrong.
 
-More information on the color encoding of resistors can be found in the [Electronic color code Wikipedia article](https://en.wikipedia.org/wiki/Electronic_color_code)
+More information on the color encoding of resistors can be found in the [Electronic color code Wikipedia article](https://en.wikipedia.org/wiki/Electronic_color_code).
+
+Create the means to look up the numerical value associated with a particular color band.
```

Comment:
Maybe move this line between the "Mnemonic" and "More information" lines? I feel like the "More information" line should probably be the last line.